### PR TITLE
Add Binary Media Types support to CloudFormation Template

### PR DESCRIPTION
**Pull request by nathanielks**
We no longer have to manually do this! Woohoo!

### CODE REVIEW

**Comment by nathanielks** [comment]
I'm testing this on a stack to verify. I'll request a review once I confirm!

File: cloudformation-template.json [hunk 1]
```diff
@@ -37,7 +37,8 @@
 		"ApiGatewayRestApi": {
 			"Type": "AWS::ApiGateway::RestApi",
 			"Properties": {
-				"Name" : "Tachyon"
+				"Name" : "Tachyon",
+				"BinaryMediaTypes": [ "image~1*" ]
```
Comment (joehoyle) [inline cloudformation-template.json]: I'm not sure what this syntax is, I thought it would be `image/*` or sometjing

Comment (rmccue) [reply cloudformation-template.json]: [Escaped slash](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-restapi.html#cfn-apigateway-restapi-binarymediatypes):
> Slashes must be escaped with `~1`. For example, `image/png` would be `image~1png` in the `BinaryMediaTypes` list.

Comment (joehoyle) [reply cloudformation-template.json]: Cray cray

**Pull request merged**